Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/program/model/util/AddressLabelInfo.java

```diff
@@ -112,12 +112,6 @@ public ProcessorSymbolType getProcessorSymbolType() {
 		return processorSymbolType;
 	}
 
-	/**
-	 * Compares this object with the specified object for order.  Returns a
-	 * negative integer, zero, or a positive integer as this object is less
-	 * than, equal to, or greater than the specified object.
-	 * @param o the object to compare with this one
-	 */
```

Comment:
This isn't adding anything to the documentation for compareTo. Its essentially just saying the same thing so I removed it.